### DISCUSSION TITLE
Issue form: Need as enterprise the required field option in internal repos

### DISCUSSION THREAD

**Discussion by Timmmy-nlb** [community/community#51500] [Enterprise] [answered]
### Select Topic Area

Product Feedback

### Body

We as a company (enterprise cloud) would like to use the issue forms more, but the current restriction that fields in internal repos cannot be set as required is really incomprehensible.
Why is there a distinction between public and internal/private?
If someone is not able to create forms and they are rejected by the user, that's bad luck, but that's why you can simply ban this option!

How should a company be able to set internal specifications for requests? Which company has no requirements in forms?
Sorry, but no way.

`Note: The required field key is only supported in public repositories. In private and internal repositories, all fields are optional.`
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

**Comment by gualterandre** [comment]
we so much need this as well!!!

**Comment by vwnj84** [comment]
Required fields is *required* by compliance.

**Comment by KingOfTac** [comment]
We have need of this as well for capturing required information during internal requests that _should_ be required. This would reduce the amount of back and forth needed to gather information because a field was left blank.

**Comment by twisterdotcom** [comment]
Added my thoughts on this here: https://github.com/orgs/community/discussions/4264#discussioncomment-6071942, but I agree we really need this.

**Comment by JonasDevDE** [comment]
Really? This is not done yet? After nearly 2 years?

**Comment by Timmmy-nlb** [comment, marked as answer]
Good News: "required" is now working in Internal Repos!
https://github.com/orgs/community/discussions/148713#discussioncomment-12226996
